haas/ext/switches/nexus.py

```diff
@@ -105,12 +105,6 @@ def connect(switch):
         console.expect('Password: ')
         console.sendline(switch.password)
 
-        console = pexpect.spawn('telnet ' + switch.hostname)
```

Comment:
I came across the redundant code during my recent cloud computing mentor meeting.